.vib/kibana/cypress/cypress/integration/kibana_spec.js

```diff
@@ -1,68 +1,58 @@
 /// <reference types="cypress" />
 
-import {
-  random,
-  closeThePopups,
-  skipTheWelcomeScreen
-} from './utils'
+import { random, closeThePopups, skipTheWelcomeScreen } from './utils';
 
 before(() => {
-  skipTheWelcomeScreen()
-})
+  skipTheWelcomeScreen();
+});
 
 it('allows uploading a file', () => {
-  cy.get('a[data-test-subj="uploadFile"]').click()
-  cy.get(
-    'input#filePicker',
-  ).selectFile('cypress/fixtures/test-data-to-import.ndjson', {
-    force: true
-  });
-  cy.get('[data-test-subj="dataVisualizerFileOpenImportPageButton"]')
-    .click();
+  cy.get('a[data-test-subj="uploadFile"]').click();
+  cy.get('input#filePicker').selectFile(
+    'cypress/fixtures/test-data-to-import.ndjson',
+    {
+      force: true,
+    }
+  );
+  cy.get('[data-test-subj="dataVisualizerFileOpenImportPageButton"]').click();
   cy.fixture('testdata').then((td) => {
     cy.get('[data-test-subj="dataVisualizerFileIndexNameInput"]').type(
-      `${td.testIndexName}.${random}`,
-    )
-  })
-  cy.get('[data-test-subj="dataVisualizerFileImportButton"]')
-    .click();
-  cy.get('[title*="Index created is complete"]')
-    .scrollIntoView()
-    .should('be.visible');
-  cy.get('[title*="Data uploaded is complete"')
-    .scrollIntoView()
-    .should('be.visible');
-  cy.get('[title*="Index pattern created is complete"]').should(
-    'be.visible',
-  );
+      `${td.testIndexName}.${random}`
+    );
+  });
+  cy.get('[data-test-subj="dataVisualizerFileImportButton"]').click();
+  cy.get('[title*="Index created"]').scrollIntoView().should('be.visible');
+  cy.get('[title*="Data uploaded"').scrollIntoView().should('be.visible');
+  cy.get('[title*="Index created"]').should('be.visible');
```

Comment:
This assertion is the same one as line `24`. Looking at the old code, maybe you meant `Index pattern created`  instead?

Reply:
Good catch, it should actually say `Data view created`.
I've modified the assertions and removed the extra `should('be.visible')` assertions. Please check now.